Formula/wget2.rb

```diff
@@ -0,0 +1,67 @@
+class Wget2 < Formula
+  desc "Multithreaded metalink/file/website downloader"
+  homepage "https://gitlab.com/gnuwget/wget2/"
+  url "https://ftp.gnu.org/gnu/wget/wget2-2.0.0.tar.gz"
+  sha256 "4fe2fba0abb653ecc1cc180bea7f04212c17e8fe05c85aaac8baeac4cd241544"
+  license "GPL-3.0-or-later"
```

Comment:
```suggestion
  license all_of: ["GPL-3.0-or-later", "LGPL-3.0-or-later"]
```

Reply:
Will fix in next commit (address some `audit` issue too)